crates/uv/src/commands/run.rs

```diff
@@ -176,7 +176,7 @@ pub(crate) async fn run_to_completion(mut handle: Child) -> anyhow::Result<ExitS
                     };
 
                     // We unconditionally forward SIGTERM to the child process; unlike SIGINT, this
-                    // isn't usually handled by the shell and in cases like
+                    // isn't usually handled by the terminal and in cases like
```

Comment:
Not new in this PR, but it seems like this sentence got cut in half?

Reply:
Thanks!